src/SFA.DAS.QnA.Application/Commands/SetPageAnswers/SetAnswersBase.cs

```diff
@@ -120,10 +120,28 @@ protected Next GetNextAction(Page page, List<Answer> answers, ApplicationSection
                             var applicationData = JObject.Parse(application.ApplicationData);
                             var questionTag = applicationData[condition.QuestionTag];
 
-                            if (questionTag == null || questionTag.Value<string>() != condition.MustEqual)
+
+                            if (questionTag == null)
                             {
                                 someConditionsNotSatisfied = true;
                             }
+                            else
+                            {
+                                if (!string.IsNullOrEmpty(condition.MustEqual) && questionTag.Value<string>() != condition.MustEqual)
+                                {
+                                    someConditionsNotSatisfied = true;
+                                }
+
+                                if (!string.IsNullOrEmpty(condition.Contains))
```

Comment:
Can you add a couple of unit tests into SetPageAnswersTests

SetAnswersBase is used in quite a few places